shopsafe-backend/src/main/java/com/google/sps/data/StoreDatastoreHandler.java

```diff
@@ -17,6 +17,7 @@
 import java.io.IOException;
 import java.util.List;
 import java.util.Map;
+import java.util.ArrayList;
```

Comment:
Just wondering, why was this needed?

Reply:
yeah, leftover code from debugging ill remove it in the next commit

Reply:
No problem!